src/quo/components/list_items/token_value/schema.cljs

```diff
@@ -0,0 +1,23 @@
+(ns quo.components.list-items.token-value.schema)
+
+(def ^private ?values
+  [:map
+   [:crypto-value :string]
+   [:fiat-value :string]
+   [:percentage-change {:optional true} :string]
+   [:fiat-change {:optional true} :string]])
+
+(def ^private ?schema
```

Comment:
Just saw there's a minor typo, missing colon to make it metadata. There are many instances of this typo in the PR. Sorry I missed this in previous review pass.

Reply:
Thanks for noticing!